File: original interface.py
import controller as controller
import cv2
import mediapipe as mp
import numpy as np
from tkinter import messagebox 
from matplotlib import pyplot as plt
import subprocess
try:
    import Tkinter as tk
except:
    import tkinter as tk
from tkinter import *
import os
from PIL import ImageTk, Image
from tkinter import filedialog
from tkinter import *
import webbrowser
import PIL.Image

ws = Tk()
ws.title('try_btn')
new = 1
url = 'https://www.canva.com/design/DAE7Y9mhbWI/PBOKmoB1WWus2jsTu12KWA/view?utm_content=DAE7Y9mhbWI&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton'
ws.geometry('354x633')

bg = PhotoImage(file = "canuseinterface.PNG")

canvas = Canvas(
    ws,
    width=500,
    height=400
)
canvas.create_image(0,0,anchor="nw",image=bg)
canvas.pack(fill='both', expand=True)

def nutrients():
    subprocess.call(["python", "d_project.py"])
    #os.system('d_project.py')
def recipe():
    subprocess.call(["python", "recipe_request.py"])
    #os.system('recipe_request.py')
def sports():
    subprocess.call(["python", "register.py"])


def openbrowser():
    webbrowser.open(url, new=new)

#canvas.create_image(0, 0, anchor="nw", image=bg)
#########recipe#########
recipe_btn = Button(ws,text='    ',bg ="#4ED1C1" ,command=recipe,width=6,height=2,borderwidth=0,font=('arial', 18))
btn_canvas = canvas.create_window(72,360,anchor="nw",window=recipe_btn)
#########recipe#########

########nutrients######

nutrient_btn = Button(ws,text='    ',bg ="#F0657A" ,command=nutrients,width=6,height=2,borderwidth=0,font=('arial', 18))
btn_canvas = canvas.create_window(240,360,anchor="nw",window=nutrient_btn)
########nutrients######

##########sport######
# sport_btn = Button(ws,text='    ',bg ="#FFD386" ,command=recipy,width=6,height=2,borderwidth=0,font=('arial', 18))
# btn_canvas = canvas.create_window(72,498,anchor="nw",window=sport_btn)
#sport_btn = Button(ws,text='    ',bg ="#FFD386" ,command=quitwindows())
sport_btn = Button(ws,text='    ',bg ="#FFD386" ,command=sports,width=6,height=2,borderwidth=0,font=('arial', 18))
btn_canvas = canvas.create_window(72,498,anchor="nw",window=sport_btn)


#squart_btn = Button(chooseme, text="Squart",command = square)
##########sport######
#########project link####################
project_link_btn = Button(ws,text='    ',bg ="#86DDFF" ,command=openbrowser,width=6,height=2,borderwidth=0,font=('arial', 18))
btn_canvas = canvas.create_window(240,498,anchor="nw",window=project_link_btn)
#########project link####################




ws.mainloop()

#cv2.destroyAllWindows()

File: original_interface.java
import java.awt.Canvas;
import java.awt.Image;
import java.awt.Toolkit;
import java.awt.event.ActionEvent;
import java.awt.event.ActionListener;
import java.io.IOException;

import javax.swing.ImageIcon;
import javax.swing.JButton;
import javax.swing.JFrame;
import javax.swing.JLabel;

public class Main {
    public static void main(String[] args) throws IOException {
        JFrame frame = new JFrame("try_btn");
        frame.setSize(354, 633);
        frame.setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);

        ImageIcon bgIcon = new ImageIcon("canuseinterface.PNG");
        Image bgImage = bgIcon.getImage();
        Image scaledBgImage = bgImage.getScaledInstance(500, 400, Image.SCALE_SMOOTH);
        ImageIcon scaledBgIcon = new ImageIcon(scaledBgImage);
        JLabel bgLabel = new JLabel(scaledBgIcon);
        bgLabel.setBounds(0, 0, 500, 400);

        frame.add(bgLabel);

        JButton recipeBtn = new JButton();
        recipeBtn.setBounds(72, 360, 60, 60);
        recipeBtn.setBorderPainted(false);
        recipeBtn.setFocusPainted(false);
        recipeBtn.setContentAreaFilled(false);
        recipeBtn.addActionListener(new ActionListener() {
            public void actionPerformed(ActionEvent e) {
                try {
                    Runtime.getRuntime().exec("python recipe_request.py");
                } catch (IOException ex) {
                    ex.printStackTrace();
                }
            }
        });

        ImageIcon recipeIcon = new ImageIcon("recipe_icon.png");
        Image recipeImage = recipeIcon.getImage();
        Image scaledRecipeImage = recipeImage.getScaledInstance(60, 60, Image.SCALE_SMOOTH);
        ImageIcon scaledRecipeIcon = new ImageIcon(scaledRecipeImage);
        recipeBtn.setIcon(scaledRecipeIcon);

        frame.add(recipeBtn);

        JButton nutrientBtn = new JButton();
        nutrientBtn.setBounds(240, 360, 60, 60);
        nutrientBtn.setBorderPainted(false);
        nutrientBtn.setFocusPainted(false);
        nutrientBtn.setContentAreaFilled(false);
        nutrientBtn.addActionListener(new ActionListener() {
            public void actionPerformed(ActionEvent e) {
                try {
                    Runtime.getRuntime().exec("python d_project.py");
                } catch (IOException ex) {
                    ex.printStackTrace();
                }
            }
        });

        ImageIcon nutrientIcon = new ImageIcon("nutrient_icon.png");
        Image nutrientImage = nutrientIcon.getImage();
        Image scaledNutrientImage = nutrientImage.getScaledInstance(60, 60, Image.SCALE_SMOOTH);
        ImageIcon scaledNutrientIcon = new ImageIcon(scaledNutrientImage);
        nutrientBtn.setIcon(scaledNutrientIcon);

        frame.add(nutrientBtn);

        JButton sportBtn = new JButton();
        sportBtn.setBounds(72, 498, 60, 60);
        sportBtn.setBorderPainted(false);
        sportBtn.setFocusPainted(false);
        sportBtn.setContentAreaFilled(false);
        sportBtn.addActionListener(new ActionListener() {
            public void actionPerformed(ActionEvent e) {
                try {
                    Runtime.getRuntime().exec("python register.py");
                } catch (IOException ex) {
                    ex.printStackTrace();
                }
            }
        });

        ImageIcon sportIcon = new ImageIcon("sport_icon.png");
        Image sportImage = sportIcon.getImage();
        Image scaledSportImage = sportImage.getScaledInstance(60, 60, Image.SCALE_SMOOTH);
        ImageIcon scaledSportIcon = new ImageIcon(scaledSportImage);
        sportBtn.setIcon(scaledSportIcon);

        frame.add(sportBtn);

        JButton projectLinkBtn = new JButton();
        projectLinkBtn.setBounds(240, 498, 60, 60);
        projectLinkBtn.setBorderPainted(false);
        projectLinkBtn.setFocusPainted(false);
        projectLinkBtn.setContentAreaFilled(false);
        projectLinkBtn.addActionListener(new ActionListener() {
            public void actionPerformed(ActionEvent e) {
                String url = "https://www.canva.com/design/DAE7Y9mhbWI/PBOKmoB1WWus2jsTu12KWA/view?utm_content=DAE7Y9mhbWI&utm_campaign=designshare&utm_medium=link2&utm_source=sharebutton";
                try {
                    java.awt.Desktop.getDesktop().browse(java.net.URI.create(url));
                } catch (IOException ex) {
                    ex.printStackTrace();
                }
            }
        });

        ImageIcon projectLinkIcon = new ImageIcon("project_link_icon.png");
        Image projectLinkImage = projectLinkIcon.getImage();
        Image scaledProjectLinkImage = projectLinkImage.getScaledInstance(60, 60, Image.SCALE_SMOOTH);
        ImageIcon scaledProjectLinkIcon = new ImageIcon(scaledProjectLinkImage);
        projectLinkBtn.setIcon(scaledProjectLinkIcon);

        frame.add(projectLinkBtn);

        frame.setLayout(null);
        frame.setVisible(true);
    }
}
